components/proxy/src/main/java/com/hotels/styx/metrics/reporting/graphite/GraphiteConfig.java

```diff
@@ -62,16 +65,23 @@ public long intervalMillis() {
         return intervalMillis;
     }
 
+    @JsonProperty("enabled")
+    public boolean enabled() {
+        return enabled;
+    }
+
     @Override
     public String toString() {
-        return new StringBuilder(96)
+        return new StringBuilder()
```

Comment:
NBD: I realise that this is just how you found it but there is no advantage over string concatenation to using `StringBuilder`.

Reply:
I agree, will simplify this implementation.